### PR TITLE
EdgeLB v1.1.0 release notes

### PR DESCRIPTION
**Pull request by vespian**
## Description
This PR adds release notes for EdgeLB v1.1.0 plus some minor improvements to the existing docs.

**Please do not merge until EdgeLB v1.1.0 is released**

## Related Jiras
* https://jira.mesosphere.com/browse/DCOS-25625 `edgelb: upgrade HAProxy to version 1.8`
* https://jira.mesosphere.com/browse/DCOS-38061 `[Edge-lb]: Provide operator methods to update instances with long running connections`
* https://jira.mesosphere.com/browse/COPS-3463 `Timeout kill haproxy if it takes longer than X to reload`
* https://jira.mesosphere.com/browse/COPS-3462 `EdgeLB should be able to re-load even if one of the backends have long running connections`
* https://jira.mesosphere.com/browse/DCOS-37564 `edgelb: Iptables rules removal should be attempted only once.`
* https://jira.mesosphere.com/browse/COPS-3166 `Possible Recommendation/Bug Resolution for ELB Performance`
* https://jira.mesosphere.com/browse/DCOS-39135 `Method to update edge-lb instances with long running connections`

## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/edge-lb/1.0/release-notes/index.md [hunk 1]
```diff
@@ -10,11 +10,57 @@ enterprise: false
 
 Release notes for Edge-LB.
 
+# v1.1.0
+
+## Noteworthy changes:
+
+- Updates HaProxy used by the pool servers to 1.8.12 from 1.7.6, please check the release notes for details
+- Pool server reloads are no longer blocked by persistent connections.
```
Comment (masonse) [inline pages/services/edge-lb/1.0/release-notes/index.md]: Please link the text "release-notes" to the actual HAProxy Release notes.

File: pages/services/edge-lb/1.0/release-notes/index.md [hunk 2]
```diff
@@ -10,11 +10,57 @@ enterprise: false
...
+- Stability, debuggability and reliability improvements in the pool server code.
+- The size of the pool container was reduced to 100MB from ~250
```
Comment (masonse) [inline pages/services/edge-lb/1.0/release-notes/index.md]: Missing units in "~250"

File: pages/services/edge-lb/1.0/release-notes/index.md [hunk 3]
```diff
@@ -10,11 +10,57 @@ enterprise: false
...
+- Transition to Master/Worker mode in HaProxy running on the pool server. If custom configuration templates are used, then they must be updated. Namely:
+  - template must not specify the demon option
```
Comment (masonse) [inline pages/services/edge-lb/1.0/release-notes/index.md]: "demon" or "daemon"

File: pages/services/edge-lb/1.0/release-notes/index.md [hunk 4]
```diff
@@ -10,11 +10,57 @@ enterprise: false
...
+  - template must specify the `expose-fd listeners` option in the stats socket
+- EdgeLb now uses the default edgelb-pool CLI packages from DC/OS SDK. They do not support as of time being the `version` subcommand.
```
Comment (masonse) [inline pages/services/edge-lb/1.0/release-notes/index.md]: Clarify the word "they".

**Comment by masonse** [comment]
@vespian Can we add a date to the release notes corresponding to the date of the release of these?

**Comment by vespian** [comment]
@masonse  Sure, will fix it once we release 1.1

**Comment by vespian** [comment]
@SarahStegall @masonse  I have added one extra commit:

https://github.com/mesosphere/dcos-docs-site/pull/1239/commits/d89e612a8b985d786a03683ef3db1805dc88d1ac

The EdgeLB changes are merged, I will be doing the release today. Let's please get the release notes PR merged.

**Pull request merged**